public/components/event_analytics/explorer/log_explorer.scss

```diff
@@ -25,11 +25,17 @@
   }
 }
 
+.logs-eui-icon {
+  display: inline-block;
+  vertical-align: text-bottom;
```

Comment:
why aligned to text-bottom?

Reply:
The icon and text were not aligned in the same line so doing that aligned them together, this is how it looked before
<img width="949" alt="image" src="https://github.com/opensearch-project/dashboards-observability/assets/107440906/4bfc11cb-332f-4903-9fb7-303947025860">

Reply:
had a discussion with Eric and changed the icon to use button

Reply:
Hey Sumuk, could you attach a screenshot in description for new look and feel?

Reply:
<img width="939" alt="image" src="https://github.com/opensearch-project/dashboards-observability/assets/107440906/099b73d1-adb9-4059-8008-ac18a1397699">